### DISCUSSION TITLE
Suggestion to make sure versions that are not marked with "R" are marked with Pre-release in Github.

### DISCUSSION THREAD

**Discussion by Katerlad** [FirstGearGames/FishNet#487] [Documentation]
For consistency I was hoping I could convince you to mark all release versions with a R, and all non Release Versions as Pre-Release.

This makes it where your "Latest" version will always be a R version, and if people want to install pre-releases they still can, for experimental testing.

![image](https://github.com/FirstGearGames/FishNet/assets/54480322/e9ab1792-83c1-4a89-9aa8-31bcdaa51112)

**Comment by Katerlad** [comment]
It is just confusing to know what is being marked as Stable Release, and what is being marked as Pre-Release/Experimental
![image](https://github.com/FirstGearGames/FishNet/assets/54480322/3e9f4dc5-a887-4fcf-adc4-ecc89e89db18)

**Comment by FirstGearGames** [comment]
This has been done.
